gulpfile.js

```diff
@@ -59,31 +85,30 @@ gulp.task('package-resources', ['default'], () => {
   ]);
 });
 
-const browserifyCb = (cb) => {
-  return (err, stdout, stderr) => {
-    console.log(stdout);
-    console.error(stderr);
-    cb(err);
-  };
-};
-
-gulp.task('package-render', ['default'], (cb) => {
-  exec('browserify --node -e build/index.js -o ./package/index.js --im', browserifyCb(cb));
+gulp.task('package-render', ['default', 'package-init'], () => {
+  browserify('build/index.js', browserifyOpts)
+    .bundle()
+    .pipe(fs.createWriteStream('package/index.js'));
 });
 
-gulp.task('package-main', ['default'], (cb) => {
-  exec('browserify --node --igv=false -e build/electron-app.js -o ./package/electron-app.js --im', browserifyCb(cb));
+gulp.task('package-main', ['default', 'package-init'], () => {
+  browserify('build/electron-app.js', browserifyOpts)
+    .bundle()
+    .pipe(fs.createWriteStream('package/electron-app.js'));
 });
 
 gulp.task('package', ['package-resources', 'package-render', 'package-main'], (done) => {
+  const manifest = JSON.parse(fs.readFileSync('package.json', 'utf8'));
+  const electronVersion = manifest.devDependencies['electron-prebuilt'];
```

Comment:
Also, wrt carets, `electron` shouldn't use carets. The deployed version (specified in gulp) _has_ to be an explicit version. It's a good idea to use the same version for development as deployment, so might as well keep it as a single version.

Besides, there's been breakage in "bugfixes" for electron before (the `object-registry` "object of undefined" thinger)